LICENSE.md

```diff
@@ -0,0 +1,6 @@
+This app is based on the game [Cards Against Humanity](https://cardsagainsthumanity.com/)
+They say this:
+
+Cards Against Humanity is available under a [Creative Commons BY-NC-SA 2.0 license](https://creativecommons.org/licenses/by-nc-sa/2.0/legalcode). That means you can use, remix, and share the game for free, but you can’t sell it without our permission. Please do not steal our name or we will smash you
+
+This also means that this app is available under the same license. Enjoy.
```

Comment:
I dunno??? It specifically says on the cards against humanity page this:
Do I have your permission to make some crappy Cards Against Humanity thing like an app?
Cards Against Humanity is available under a BY-NC-SA 2.0 Creative Commons license. That means you can use our content to make whatever, but you have to give us credit, you can’t profit from the use of our content (this means ad revenue is not allowed), and you have to share whatever you make in the same way we share it (this means you can’t submit our content to any app store). We own the name "Cards Against Humanity," so you have to call your crappy thing something else.